src/main/java/io/confluent/connect/jdbc/util/TableDefinitions.java

```diff
@@ -83,8 +83,10 @@ public TableDefinition refresh(
       TableId tableId
   ) throws SQLException {
     TableDefinition dbTable = dialect.describeTable(connection, tableId);
-    log.info("Refreshing metadata for table {} to {}", tableId, dbTable);
-    cache.put(dbTable.id(), dbTable);
+    if (dbTable != null) {
+      log.info("Refreshing metadata for table {} to {}", tableId, dbTable);
+      cache.put(dbTable.id(), dbTable);
+    }
```

Comment:
should we log a warn in an else block here saying that the task failed to describe the table? It's hard to follow "absence of logs" when these connectors are typically running against a large number of tables.

Reply:
It looks like some of the call-sites log the warning based on the return value, but some don't.
I'll add the log message.
```suggestion
    } else {
      log.warn("Failed to refresh metadata for table {}", tableId);
    }
```